### ISSUE TITLE
Low classification accuracy on Endometrial POLE experiment reproduction

### ISSUE THREAD

**Issue by CarolRameder** [djimenezsanchez/NaroNet#8]
Hello!

I tried to reproduce the experiment described in Section 3.2. I used the same hyperparameter setup presented in Supplementary Table 3. The PCL component yielded 61.87% contrast accuracy compared to the 81.11% reported in the paper.

![PCA_result](https://github.com/user-attachments/assets/3e004191-9afb-4b7f-b15f-ae3406f43d15)

The only modification was to remove 5 images with missing labels from the dataset and use 331 instead of 336. The results can be seen in the table below. I repeated the NaroNet training with the same embeddings 5 times to analyze how the results vary.

![results](https://github.com/user-attachments/assets/3e6f8bd8-f96e-4d73-af4b-20436820f61c)

Do you have any idea why this happens? 
Thanks in advance!

**Comment by TechnoBear1000** [comment]
You can try danijimnzs@gmail.com
